NOLF2/ObjectDLL/ObjectShared/WorldModel.cpp

```diff
@@ -403,6 +403,15 @@ void WorldModel::ReadProps( ObjectCreateStruct *pOCS )
 		if( genProp.m_String[0] )
 		{
 			m_hstrAttachments = g_pLTServer->CreateString( genProp.m_String );
+			//HACK: For some reason, AirPlane_Prop00.Propeller's attachment target is
+			//itself, which fails an assertion in Lithtech's om_CreateAttachment function
+			//when loading the first map in Siberia. Disallow it here.
+			const char *pAttachments = g_pLTServer->GetStringData(m_hstrAttachments);
+			if (!strcmp(pOCS->m_Name, "AirPlane_Prop00.Propeller")
```

Comment:
if we could drop naming the propeller object directly
then this hack would be a bit more generic... and a design safety feature preventing direct attachment cycles